osu.Game.Tests/Visual/Gameplay/TestSceneSongProgress.cs

```diff
@@ -147,15 +84,29 @@ protected override void Update()
             framedClock.ProcessFrame();
         }
 
-        private class TestSongProgressGraph : SongProgressGraph
+        protected override Drawable CreateDefaultImplementation()
         {
-            public int CreationCount { get; private set; }
+            defaultProgress = new DefaultSongProgress
+            {
+                RelativeSizeAxes = Axes.X,
+                Anchor = Anchor.BottomLeft,
+                Origin = Anchor.BottomLeft,
+            };
+
+            progresses.Add(defaultProgress);
+            return defaultProgress;
```

Comment:
Use `this.ChildrenOfType<DefaultSongProgress>().Single()` and `this.ChildrenOfType<SongProgress>().ForEach` rather than storing references.